openhands/utils/prompt.py

```diff
@@ -63,6 +63,20 @@ def _load_template(self, template_name: str) -> Template:
         with open(template_path, 'r') as file:
             return Template(file.read())
 
+    def extend_system_prompt(self, instruction: str) -> None:
+        """
+        Extends the system prompt.
+
+        Parameters:
+            instruction: The instruction to add to the system prompt
+        """
+
+        modified_template = self.system_template.render(
+            additional_instruction=instruction
+        )
+
+        self.system_template = Template(modified_template)
```

Comment:
This probably won't work as expected if we add more templating. Instead, you want to set `self.additional_instructions` (I'd call it `runtime_info` to be more explicit), and then pass that into the `render()` in `get_system_message()` below

Reply:
Something like this?:
```py
bullet_points = "\n".join(f"* {instruction}" for instruction in self.additional_instructions)
return self.user_template.render(additional_instructions=bullet_points).strip()
```

I can't find any reference to `runtime_info`, where can I find it?